docs/management.md

```diff
@@ -48,16 +48,22 @@ This guide covers:
  * How to [enable HTTPS for management UI](#single-listener-https) and its underlying API
  * How this plugin [operates in multi-node clusters](#clustering)
  * How to [disable metric collection](#disable-stats) to use [Prometheus](./prometheus) exclusively for monitoring
- * [OAuth 2](#oauth2-authentication) support
+ * [OAuth 2](#oauth2-authentication) support:
+    * [Basic Configuration](#basic-configuration)
+    * [Configure client secret](#configure-client-secret)
+    * [Allow Basic and OAuth 2 authentication for Management HTTP API](#allow-basic-auth-for-http-api)
+    * [Allow Basic and OAuth 2 authentication for Management UI](#allow-basic-auth-for-mgt-ui)
```

Comment:
Are the procedures above optional? if so how about these as titles: Optional: "Set up Basic and OAuth 2 authentication for the Management HTTP API" - apply a similar heading to the 2nd one.

Reply:
see commit  a716027017faf9cb1a711c4b3e6a111751205532